triton-server/src/contact-service.ts

```diff
@@ -173,7 +177,11 @@ function getCredentialMessageFor(contact: Contact) {
         <br/>
         Endpoint: <b>mcgilluniversity#genomecentre-lims</b><br/>
         Username: <b>${contact.project_id}</b><br/>
-        Password: <b>${contact.depth ?? "ERROR COULD NOT GENERATE PASSWORD"}</b><br/><br/>
+        Password: <b>${contact.depth ?? "ERROR COULD NOT FIND PASSWORD"}</b><br/>
```

Comment:
So if the user does a password reset and it fails for some reason the client get sent, as password, an error message ? Do we even get a notification in that situation ? I would personally not send the client anything in that case and i would forward the username to either us or IT so we can fix this behind the scene. Not sure this does any good to send the user this email.

Reply:
This issue has been rebased out of this branch.